apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -35,6 +36,63 @@ export interface IInstallPackageInDirectoryOptions {
   suppressOutput?: boolean;
 }
 
+export interface ILifecycleCommandOptions {
+  /**
+   * Working directory for running the command
+   */
+  workingDirectory: string;
+
+  /**
+   * The folder containing a local .npmrc, which will be used for the INIT_CWD environment variable
+   */
+  initCwd: string;
+
+  /**
+   * If true, suppress the process's output, but if there is a nonzero exit code then print stderr
+   */
+  handleOutput: boolean;
+
+  /**
+   * Options for what should be added to the PATH variable
+   */
+  environmentPathOptions: Partial<IEnvironmentPathOptions>;
```

Comment:
`Partial<...>` is needless complexity.  All usages of `IEnvironmentPathOptions` are partial, so just make the interface members optional